app/controllers/items_controller.rb

```diff
@@ -693,16 +692,16 @@ def save_and_reindex
   # Permissions
 
   # check that the user can carry out this item modification
-  def forbid_modify
-    return true if current_user.is_admin || @object.can_manage_content?(current_user.roles(@apo))
-    render :status => :forbidden, :text => 'forbidden'
-    false
+  def authorize_manage_obj!
+    authorize! :manage_item, @object
   end
```

Comment:
Thanks, `manage_item` vs `manage_content` was a mistake on my part. I'm interested in understanding more about when `is_manager` implies `manage_content` and when it does not. Maybe there's something deeper going on. @LynnMcRae ?